tests/plotting/conftest.py

```diff
@@ -7,11 +7,17 @@
 import pytest
 
 import pyvista
+from pyvista.plotting import system_supports_plotting
 
 # these are set here because we only need them for plotting tests
 pyvista.global_theme.load_theme(pyvista.themes._TestingTheme())
 pyvista.OFF_SCREEN = True
 
+# Configure skip_plotting marker
+pytest.mark.skip_plotting = pytest.mark.skipif(
```

Comment:
This (slapping a new name onto `pytest.mark`) seems really hacky, but I'm not sure what else we could do... I suspect if we define a global name here it won't be available without explicitly importing it, which would defeat the purpose of having this in `conftest.py`.

Reply:
Ah, I mimicked this implementation from elsewhere. Reading up on pytest docs now and will do this properly per https://docs.pytest.org/en/7.1.x/example/markers.html#adding-a-custom-marker-from-a-plugin